src/commands/game_commands/profile.ts

```diff
@@ -50,6 +50,115 @@ export function getRankNameByLevel(level: number): string {
     return RANK_TITLES[0].title;
 }
 
+export async function getProfileFields(requestedPlayer: Eris.User): Promise<Array<Eris.EmbedField>> {
+    const playerStats = await dbContext.kmq("player_stats")
+        .select("songs_guessed", "games_played", "first_play", "last_active")
+        .where("player_id", "=", requestedPlayer.id)
+        .first();
+
+    if (!playerStats) {
+        return [];
+    }
+
+    const songsGuessed = playerStats["songs_guessed"];
+    const gamesPlayed = playerStats["games_played"];
+    const firstPlayDateString = friendlyFormattedDate(new Date(playerStats["first_play"]));
+    const lastActiveDateString = friendlyFormattedDate(new Date(playerStats["last_active"]));
+
+    const totalPlayers = (await dbContext.kmq("player_stats")
+        .count("* as count")
+        .where("exp", ">", "0")
+        .first())["count"] as number;
+
+    const { exp, level } = (await dbContext.kmq("player_stats")
```

Comment:
Isn't the query on line 54 the same as this

Reply:
Yeah it is, I just copied over the existing implementation into this function. Should I combine the queries?

Reply:
Yes